jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/MovedContextHandler.java

```diff
@@ -13,132 +13,180 @@
 
 package org.eclipse.jetty.server.handler;
 
+import org.eclipse.jetty.http.HttpField;
 import org.eclipse.jetty.http.HttpHeader;
 import org.eclipse.jetty.http.HttpStatus;
 import org.eclipse.jetty.http.HttpURI;
+import org.eclipse.jetty.http.PreEncodedHttpField;
 import org.eclipse.jetty.server.Handler;
 import org.eclipse.jetty.server.Request;
 import org.eclipse.jetty.server.Response;
 import org.eclipse.jetty.util.Callback;
 import org.eclipse.jetty.util.URIUtil;
 
 /**
- * Moved ContextHandler.
- * This context can be used to replace a context that has changed
- * location.  Requests are redirected (either to a fixed URL or to a
- * new context base).
+ * <p>A {@code ContextHandler} with a child {@code Handler}
+ * that redirects to a configurable URI.</p>
  */
 public class MovedContextHandler extends ContextHandler
 {
-    final Redirector _redirector;
-    String _newContextURL;
-    boolean _discardPathInfo;
-    boolean _discardQuery;
-    boolean _permanent;
-    String _expires;
+    private int _statusCode = HttpStatus.SEE_OTHER_303;
```

Comment:
Should document this default value in the javadoc somewhere.

In `getStatusCode()` perhaps?